### PR TITLE
[CAMEL-15871] remove item name prefix

### PR DESCRIPTION
**Pull request by schernolyas**
- [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [ ] If you're unsure, you can format the pull request title like `[CAMEL-XXX] Fixes bug in camel-file component`, where you replace `CAMEL-XXX` with the appropriate JIRA issue.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Run `mvn clean install -Psourcecheck` in your module with source check enabled to make sure basic checks pass and there are no checkstyle violations. A more thorough check will be performed on your pull request automatically.
Below are the contribution guidelines:
https://github.com/apache/camel/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: components/camel-milo/src/main/java/org/apache/camel/component/milo/server/internal/CamelServerItem.java [hunk 1]
```diff
@@ -55,7 +55,7 @@ public CamelServerItem(final String itemId, final UaNodeContext nodeContext, fin
         this.itemId = itemId;
         this.baseNode = baseNode;
 
-        final NodeId nodeId = new NodeId(namespaceIndex, "items-" + itemId);
+        final NodeId nodeId = new NodeId(namespaceIndex, "" + itemId);
```
Comment (oscerd) [inline components/camel-milo/src/main/java/org/apache/camel/component/milo/server/internal/CamelServerItem.java]: I think you can remove the empty string concatenation.

**Review by oscerd — CHANGES_REQUESTED** [review]
Please add the Camel issue number in the commit message along with the title of the issue, for tracking purpose. Thanks

File: components/camel-milo/src/main/java/org/apache/camel/component/milo/server/internal/CamelServerItem.java (hunk 1 above)
Comment (schernolyas) [reply components/camel-milo/src/main/java/org/apache/camel/component/milo/server/internal/CamelServerItem.java]: yes, of course. it is done.

**Comment by schernolyas** [comment]
> Please add the Camel issue number in the commit message along with the title of the issue, for tracking purpose. Thanks

yes. I have appended it to last commit.

**Comment by oscerd** [comment]
This has been squashed and merged. Thanks.

**Pull request closed without merging**